api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/Anc1TimingDataConverter.java

```diff
@@ -28,10 +28,10 @@ public Object convert(Object obj) {
         }
         Obs obs = ((Obs) obj);
 
-        if (obs.getValueCoded() != null && obs.getValueCoded().equals(Dictionary.getConcept(Dictionary.YES_CIEL))) {
-            return "=UNICHAR(8730)";
+        if (obs.getValueCoded() != null && obs.getValueCoded().equals(Dictionary.getConcept("dcd695dc-30ab-102d-86b0-7a5022ba4115"))) {
```

Comment:
@solemabrothers why the change from Dictionary.YES_CIEL? The old pre-2019 reports will now be broken.

Just add new checks rather than replacing the existing ones

Reply:
Done